notebooks/mcmc.ipynb

```diff
@@ -647,9 +659,10 @@
     }
    ],
    "source": [
+    "nburn = 10000 # burnin length\n",
```

Comment:
since you're hardcoding this, the plots are showing up blank when testing with `niter = 1000`

Reply:
This would have to be changed along with 'niter' based on the previous plots. I can mention that explicitly.